packages/collector/test/tracing/logger/bunyan/controls.js

```diff
@@ -67,6 +67,33 @@ function waitUntilServerIsUp() {
   );
 }
 
+/**
+ * @param opts {{
+ *   suppressTracing: boolean,
+ *   url: string,
+ *   json: boolean,
+ *   headers: {
+ *     'X-INSTANA-L': '0' | '1',
+ *     [key:string]: any
+ *   }
+ * }} The request options
+ */
+exports.sendRequest = function sendRequest(opts = {}) {
```

Comment:
FYI: Bunyan has it's own control implementation. I didn't want to go the path and figure out why. I needed the `sendRequest` method and copied it over. Nothing I wanted to refactor in this PR.